### DISCUSSION TITLE
request node: Always "Error: Forbidden"

### DISCUSSION THREAD

**Discussion by molodeztom** [alexkn/node-red-contrib-homeconnect#27] [Q&A]
Hi I want to control my dishwasher. Done so far:
- Registered App 
- Setup home-connect-auth node with client ID and secret, Scope "Dishwasher"
- Start Authorization worked (showed an "ok") Node status green "ready"
- home-connect-request node: Tag: Status Action get-status HAID set 
I trigger the request node with a timestamp
I always get the result: "Error: Forbidden".
Granted Scopes on home-connect.com all have a tick. 
I have no idea how to proceed, any help appreciated.

**Comment by molodeztom** [comment]
No ideas to help me?

**Reply by alexkn** [reply]
No clue.
Maybe try with another scope combinations.

**Reply by alexkn** [reply]
I have published a new version with improved error logging.
It should output the [API error](https://developer.home-connect.com/docs/general/apierrors) from the response body.

I hope that helps you to find the problem.

**Reply by molodeztom** [reply]
It returns 403 Scope has not been granted or home appliance is not assigned to HC account. In the authorisation page it shows all required rights. Will look further into it. But thanks anyway for the error logging.
